<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File</class>
 <widget class="QWidget" name="File">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="2">
    <widget class="QPushButton" name="Upload_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="Download_Button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;
disabled { color: red };
</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
     <property name="autoRepeatDelay">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="AddFolder_Button">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Add Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="Delete_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="UP_BUTTON">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;  
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/icons/Up.jpg</normaloff>:/img/icons/Up.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="DOWN_BUTTON">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;  
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/icons/DOWN.jpg</normaloff>:/img/icons/DOWN.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="REFRESH_BUTTON">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;  
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/img/icons/Refresh.jpg</normaloff>:/img/icons/Refresh.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="Latest_Button">
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Latest</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="6" colspan="6">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="Photo_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Photo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="Video_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Video</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="Music_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="Document_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Document</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="Else_Button">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Comic Sans MS&quot;;
color: rgb(0, 0, 0);
border: 2px solid black; 
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Else</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>Delete_Button</zorder>
  <zorder>Download_Button</zorder>
  <zorder>AddFolder_Button</zorder>
  <zorder>Upload_Button</zorder>
  <zorder>Latest_Button</zorder>
  <zorder>Photo_Button</zorder>
  <zorder>Video_Button</zorder>
  <zorder>Music_Button</zorder>
  <zorder>Document_Button</zorder>
  <zorder>Else_Button</zorder>
  <zorder>widget</zorder>
  <zorder>UP_BUTTON</zorder>
  <zorder>DOWN_BUTTON</zorder>
  <zorder>REFRESH_BUTTON</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
